tile-reduce/index.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for tile-reduce 3.1.1
+// Project: https://github.com/mapbox/tile-reduce
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace NodeJS  {
+  interface Global {
+    mapOptions: any
+  }
+}
+
+interface startEvent {
+    (): void;
+}
+
+interface mapEvent {
+    (tile: TileReduce.Tile, workerId: number): void;
+}
+
+interface reduceEvent {
+    (result: any, tile: TileReduce.Tile): void;
+}
+
+interface endEvent {
```

Comment:
Oh, `declare global` only works inside of a module context.
To convert to a module, just remove the `declare module "tile-reduce" { }` wrapper around `export = TileReduce`.
Then `declare global` will work.
Then in the test file you can replace `global.mapOptions` with just `mapOptions`. This would eliminate the dependency on node typings.
Although, if you would rather write `global.mapOptions`, you could leave the code as is, since this looks like a node-only library.